docs/docs/themes/introduction.md

```diff
@@ -1,51 +1,31 @@
-
-If you want to avoid things like GraphQL queries for common features in a blog, you can begin with a base theme and get started building and designing components. If you're comfortable in CSS and want to adjust the styling of a blog, that's an option, too! This makes building sites with Gatsby more accessible to a much larger audience. Down the road, if your blog gets more complex or your needs change, you can gradually introduce Gatsby's powerful GraphQL features.
-
-Themes benefit starter authors since they can be shipped as libraries. This means that the functionality isn’t directly written into the project but as a dependency. Users can now seamlessly update your theme, compose them together, and even swap out one compatible theme for another.
+- Sites created from a traditional starter have basically been "ejected" from the starter -- They maintain no connection to the starter, and begin to diverge immediately. If the starter is updated later, there's no easy way to pull upstream changes into an existing project.
+- If you created multiple sites using the same starter, and later wanted to make the same update to all of those sites, you'd have to do them individually, site-by-site.
 
-## Why Does This Matter?
+## Gatsby themes
 
-Theming in Gatsby means you can now avoid the boilerplate required when setting up a new project that’s commonplace in the community. That means we can arrive at a shared convention for something like a blog, and not be required to write GraphQL or configure a plugin unless you want to. No longer do you have to reinvent the wheel for each new Gatsby site you want to build.
+Enter themes. Gatsby themes allow Gatsby site functionality to be packaged as a standalone products for others (and yourself!) to easily re-use. Using a traditional starter, all of your default configuration lives directly in your site. Using a theme, all of your default configuration lives in an npm package.
 
-It provides a foundation to build upon, and since themes can inherit from other themes you can focus on building out new abstractions or functionality. And by including [accessibility](./docs/themes/building-themes/#make-it-accessible-by-default) in your theme, you can contribute back to create a more accessible web!
+Themes solve the problems that traditional starters experience:
 
-## How Do You Customize a Theme?
+- Sites created using a Gatsby theme can adopt upstream changes to the theme -- themes are versioned packages that can be updated like any other package.
+- You can create multiple sites that consume the same theme. To make updates across those sites, you can update the central theme and bump the version in the sites (rather than spending the time to tediously update the functionality of each individual site).
```

Comment:
```suggestion
- You can create multiple sites that consume the same theme. To make updates across those sites, you can update the central theme and bump the version in the sites through `package.json` files (rather than spending the time to tediously update the functionality of each individual site).
```